mrbelvedereci/testresults/models.py

```diff
@@ -49,15 +49,11 @@ def compare_results(self, build_flows):
                     results[cls][method] = {}
     
                 for limit in result.get_limit_types():
-                    limit = limit + '_used'
-                    test_limit = 'test_' + limit
+                    test_limit = 'test_{}_used'.format( limit )
     
-                    if limit not in results[cls][method]:
-                        results[cls][method][limit] = OrderedDict()
                     if test_limit not in results[cls][method]:
                         results[cls][method][test_limit] = OrderedDict()
     
-                    results[cls][method][limit][build_flow.id] = getattr(result, limit)
```

Comment:
the tooling api test runner means we only care about the `test_` limits (the transactions between Test.startTest() and Test.startTest()).

so, the non `test_` limits will be None on both sides of the comparison.